rioting-daisies/BangazonWorkforce
# Bj computer controller

This request adds the Index, Create, Details, and Delete Functionality to the computer controller, and the views for each action. It adds a new property to the computer model to store a value if the computer has been assigned at any point. 



Fixes Issue # 7

## Type of change

Please delete options that are not relevant.



# Testing Instructions

1. Git fetch and checkout BJ-Computer-Controller
1. Run Bangazon and navigate to the computers tab.
1. hit the create new to create a new computer, check that it has been created in the index, then click details, delete, and confirm delete.
1. check that the computer has been deleted from the index and DB. 
1. Click details for the first computer and confirm there is no delete button shown. 




# Checklist:
